orion-viewer/src/main/java/universe/constellation/orion/viewer/prefs/GlobalOptions.kt

```diff
@@ -138,6 +138,9 @@ class GlobalOptions(
     val isSwapKeys: Boolean
         get() = getBooleanProperty(SWAP_KEYS, false)
 
+    val isSyncXScroll: Boolean
```

Comment:
@zhmylove here you use old Orion's approach to work with properties, it doesn't  reflect changes in setting until application restart. Please take a look on DRAW_PAGE_BORDER property below, it defines OBSERVABLE property that listens all changes in settings

Reply:
Sure, I'll update this part of code soon

Reply:
@max-kammerer Hello! I've pushed new variant. This time it respects settings change just in time